app/Listeners/SPIDEventSubscriber.php

```diff
@@ -23,7 +24,7 @@ public function onSPIDLogin(LoginEvent $event): void
         auth()->logout();
         $SPIDUser = $event->getSPIDUser();
         $user = User::findByFiscalNumber($SPIDUser->fiscalNumber);
-        if (isset($user)) {
+        if (isset($user) && $user->isNotAn(UserRole::SUPER_ADMIN)) {
```

Comment:
In questo modo, però, facendo il login con account SPID con CF corrispondente a quello di un SUPER_ADMIN, ad ogni successivo login si finisce sempre nella pagina di registrazione (anche dopo aver completato la verifica).
Al momento direi che è meglio rispetto ad andare in errore per mancanza della PA nella sessione, ma sarei per gestirlo meglio in futuro, che ne pensi?
Che ne pensi?

Reply:
Ho pushato una possibile soluzione.